interface/billing/payment_master.inc.php

```diff
@@ -259,7 +259,7 @@ function generate_list_payment_category($tag_name, $list_id, $currvalue, $title,
                 <div class="form-group">
                     <div class="col-sm-12 text-left position-override">
                         <div class="btn-group btn-group-pinch" role="group">
-                            <button onClick="javascript:return SavePayment();" class="btn btn-secondary btn-save"><?php echo xlt('Save Changes');?></button>
+                            <input type="button" onClick="javascript:return SavePayment();" class="btn btn-primary btn-save" value="<?php echo xla('Save Changes');?>"></input>
```

Comment:
I'd add an HTML comment in here that this is an exception to the styling rules we came up with

Reply:
There is nothing wrong with having inputs as buttons!
Still in this case it should stay as a button. Curious @stephenwaite  why the change?
I'd leave as was but remove the javascript: that is way old and no longer needed.

Reply:
@sjpadgett , When using input as button, unable to incorporate the fontawesome icon classes. Unless you have a solution for this issue here:
https://github.com/openemr/openemr/issues/3251

Reply:
@bradymiller A mystery for another day.:) I have enough of them currently. I've used FA with inputs many times in the past so....?

Reply:
I believe we use to have style classes to handle this:

        input[type="button"] {
            font-family: FontAwesome;
        }
etc. If not need to add because we shouldn't throw out the cat just to display an icon...

Reply:
We have a really nice icon mechanism (`btn-save` for example in above line, which will do nothing in the above `input` line) that can not work on input.
The current mechanism makes it easy to add icons to buttons and also keeps them centralized so easy to modify them in the future (for example, bs5 will have its own icon set). We all took a stab at solving the issue, and couldn't. So we threw the cat out :)
Are we losing something by converting the input to a button?